packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -238,6 +238,78 @@ export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
 
 // https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-integration-lambda-extensions-versions.html
 export const APPCONFIG_LAMBDA_LAYER_ARNS: { [key: string]: any } = {
+  '2.0.2037': {
```

Comment:
For PR reviewer: the content of this map was generated by this pytyhon script
https://github.com/aws/aws-cdk/issues/35937#issuecomment-3486997520